environs/bootstrap/bootstrap.go

```diff
@@ -513,10 +540,9 @@ func bootstrapImageMetadata(
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	// This constraint will search image metadata for all supported architectures and series.
 	imageConstraint := imagemetadata.NewImageConstraint(simplestreams.LookupParams{
 		CloudSpec: region,
-		Series:    availableTools.AllSeries(),
```

Comment:
We are getting tools later now. At this stage, we do not know what tools we have yet. So we are getting all image metadata.